packages/app-project/package.json

```diff
@@ -4,15 +4,19 @@
   "scripts": {
     "_check": "check-engines && check-dependencies",
     "_serve": "node server",
-    "build": "npm run _check && next build",
-    "dev": "npm run _check && npm run _serve",
-    "start": "npm run _check && NODE_ENV=production npm run _serve",
-    "test": "npm run _check && BABEL_ENV=test mocha || true"
+    "build": "npm run _check && PANOPTES_ENV=production next build",
+    "dev": "npm run _check && PANOPTES_ENV=production npm run _serve",
+    "test": "npm run _check && BABEL_ENV=test mocha || true",
+    "start": "npm run _check && NODE_ENV=production npm run _serve"
```

Comment:
Yes - the `start` command is used to start the production server, and `dev` starts dev. This is a nextjs convention, but we don't have to stick with it.

Reply:
All of us are used to developing against staging, so I'm not sure if that's a good idea to set the environment to production just because it's a next.js convention.